### PR TITLE
Azure ARM and Cloud: Fixed lowercase type casting

### PR DESCRIPTION
**Pull request by rezoan**

### CODE REVIEW

File: azure/cloud/Keyvault.rego [hunk 1]
```diff
@@ -34,7 +34,7 @@ azure_issue["KeyVault"] {
     resource := input.resources[_]
     lower(resource.type) == "microsoft.keyvault/vaults"
     count([c | r := input.resources[_];
-              r.type == "microsoft.keyvault/vaults/accessPolicies";
+              lower(r.type) == "microsoft.keyvault/vaults/accessPolicies";
```
Comment (farchide) [inline azure/cloud/Keyvault.rego:37]: @rezoan `Policies` should be lower case as well

File: azure/iac/KeyVault.rego [hunk 2]
```diff
@@ -34,7 +34,7 @@ azure_issue["KeyVault"] {
     resource := input.resources[_]
     lower(resource.type) == "microsoft.keyvault/vaults"
     count([c | r := input.resources[_];
-              r.type == "microsoft.keyvault/vaults/accessPolicies";
+              lower(r.type) == "microsoft.keyvault/vaults/accessPolicies";
```
Comment (farchide) [inline azure/iac/KeyVault.rego:37]: `Policies` should be lowercase

File: azure/cloud/Keyvault.rego (hunk 1 above)
Comment (rezoan) [reply azure/cloud/Keyvault.rego]: fixed

File: azure/iac/KeyVault.rego (hunk 2 above)
Comment (rezoan) [reply azure/iac/KeyVault.rego]: fixed

**Review by farchide — APPROVED** [review]

**Pull request merged**